README.md

```diff
@@ -1,49 +1,133 @@
-* [GitHub releases](https://github.com/Fabulously-Optimized/fabulously-optimized/releases)
-* Chat Reporting Helper resource pack: [CurseForge](https://www.curseforge.com/minecraft/texture-packs/chat-reporting-helper) | 
-[Modrinth](https://modrinth.com/resourcepack/chat-reporting-helper) |
-[Planet Minecraft](https://www.planetminecraft.com/texture-pack/chat-reporting-helper) (bundled in FO)
-* Fast Better Grass resource pack: [CurseForge](https://www.curseforge.com/minecraft/texture-packs/fast-better-grass) | [Modrinth](https://modrinth.com/resourcepack/fast-better-grass) | [Planet Minecraft](https://www.planetminecraft.com/texture-pack/fast-better-grass) (bundled in FO)
-</details>
+<img src="https://download.fo/icon.png" alt="FO logo" align="right" width="10%" height="10%"/>
+
+# Fabulously Optimized
+
+[![Official Website][img-website]][url-website]
+[![Read the Wiki][img-wiki]][url-wiki]
+[![Support and Contribute][img-thanks]][url-thanks]
+[![Help us Translate][img-translate]][url-translate]
+
+[![Discord][img-discord]][url-discord]
+[![GitHub][img-github]][url-github]
+[![Mastodon][img-mastodon]][url-mastodon]
+
+[![Mods list][img-mods]][file-mods]
+[![Options changed][img-options]][url-options]
+[![Resource Packs][img-rp]][url-rp]
+
+A simple Minecraft modpack focusing on performance
+and graphics enhancements.
+
+## Download
+
+[![CurseForge][img-cf]][url-cf]
+[![Modrinth][img-modrinth]][url-modrinth]
+[![Installation Guide][img-install]][url-install]
+
+> By downloading Fabulously Optimized, you have
+> acknowledged the [disclaimers][url-terms].
+
+We recommend and support five launchers, but you can also
+download the modpack from [GitHub Releases][url-app-github].
+
+All five launchers can be used offline, and they run on
+🪟 Windows, 🐧 Linux and 🍏 macOS.
+
+See this table for a comparison between the launchers:
+
+| Launcher                                      |               Tutorial               |    Install    |    Upgrade    |   Add mods    | Support us and mods |
+| --------------------------------------------- | :----------------------------------: | :-----------: | :-----------: | :-----------: | :-----------------: |
+| [CurseForge App][url-app-cf]                  |    [🔗&nbsp;Open][url-tutorial-cf]    |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |     ✅&nbsp;Yes      |
+| [Modrinth App][url-app-modrinth]              | [🔗&nbsp;Open][url-tutorial-modrinth] |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |     ✅&nbsp;Yes      |
+| [Prism Launcher][url-app-prism]               |  [🔗&nbsp;Open][url-tutorial-prism]   |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ➖&nbsp;Partially   |
+| [MultiMC][url-app-multimc]                    | [🔗&nbsp;Open][url-tutorial-multimc]  |  ✅&nbsp;Easy  | ❌&nbsp;Manual | ❌&nbsp;Manual |  ➖&nbsp;Partially   |
+| [MultiMC auto-update][url-app-mmcu]           |   [🔗&nbsp;Open][url-tutorial-mmcu]   |  ✅&nbsp;Easy  | ❌&nbsp;Manual | ❌&nbsp;Manual |  ➖&nbsp;Partially   |
```

Comment:
Is there a reason that the Upgrade tab here is set to manual? The point of auto-update is that it's easier to update. You can't update to a newer MC version automatically, so maybe that's being referred to here?

Reply:
I agree - it shouldn't be "Manual", it should be something like "within the same Minecraft version" but somehow written more shortly.

Reply:
I used a footnote.